remote_config/remote_config_resources/build.gradle

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+buildscript {
+  repositories {
+    google()
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'com.android.tools.build:gradle:3.3.3'
+    classpath 'com.google.gms:google-services:4.2.0'
+  }
+}
+allprojects {
+  repositories {
+    google()
+    mavenCentral()
+  }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+  compileOptions {
+    sourceCompatibility 1.8
+    targetCompatibility 1.8
+  }
```

Comment:
Is this necessary?

Reply:
This file is necessary in order to build the new java source files for remote config.

I'm not sure what some fields like compileOptions and compileSdkVersion do exactly, but I copied most of this file from app_check_resources and it seems similar to many other _resources gradle files in the cpp sdk.